index.js

```diff
@@ -1112,17 +1113,29 @@ class GithubScm extends Scm {
      * @return {Promise}                      Resolves to string containing contents of file
      */
     async _getFile({ scmUri, path, token, ref, scmRepo }) {
-        const lookupConfig = {
-            scmUri,
-            token
-        };
+        let fullPath = path;
```

Comment:
can we also override the provider in jobs? If yes, can we override the properties individually?

Reply:
I'll look into it in the models/config-parser repos
I think we'll need to implement something similar to this to allow merging of individual properties under provider: 
https://docs.screwdriver.cd/user-guide/templates#merging-with-shared-steps